api/read/get_roc_contacts/index.md

```diff
@@ -26,31 +26,31 @@ This method optionally supports paging, to iterate over all results,
 ```xml
 <?xml version="1.0" encoding="UTF-8"?>
 <results>
-  <ROC ID="4" ROC_NAME="EGI.eu">
-    <ROCNAME>EGI.eu</ROCNAME>
-    <MAIL_CONTACT>example@egi.eu</MAIL_CONTACT>
-    <GOCDB_PORTAL_URL>/portal/index.php?Page_Type=NGI&id=4</GOCDB_PORTAL_URL>
-    <CONTACT USER_ID="406G0" PRIMARY_KEY="406G0">
-      <FORENAME>Example</FORENAME>
-      <SURNAME>User</SURNAME>
+  <ROC ID="11463" ROC_NAME="NGI_SI">
+    <ROCNAME>NGI_SI</ROCNAME>
+    <MAIL_CONTACT>ngi-si-contact@swing-grid.si</MAIL_CONTACT>
+    <GOCDB_PORTAL_URL>/portal/index.php?Page_Type=NGI&id=11463</GOCDB_PORTAL_URL>
```

Comment:
Not actually sure how important it is to escape the ampersands, but in some places they are and GitHub is highlighting them in red.

Reply:
They should be escaped as GOCDB escapes them in it's API output (and indeed an unescaped `&` is invalid XML it seems).

Reply:
I have escaped them in my latest commit, thanks.